src/styles/blocks/footer.scss

```
@@ -0,0 +1,167 @@
+.footer {
+  background-color: $greenTextColor;
+  padding: 30px 0;
+  opacity: 50%;
```

```suggestion
  opacity: 0.5;
```
opacity should be from 0 to 1. opacity in % brawser set to 1%.